tests/health_checks/NDv4/ecc-errors.sh

```diff
@@ -73,18 +84,98 @@ uncorrectable_v=( ${uncorrectable} )
 
 #Identify any GPUs that have experienced more row remaps than allowed.
 for i in $(seq 0 $((ngpus-1))); do
-	val=$((correctable_v[i] + uncorrectable_v[i]))
-	if [ $val -gt 511 ]; then
+	val=$((uncorrectable_v[i]))
+	if [ $val -gt 512 ]; then
+		pass=0
+		echo "GPU $i has $val uncorrectable row remap errors."
+	fi
+done
+
+uncorrectableVS=$(echo "$ECC_errors" | grep Volatile -A 4 \
```

Comment:
uncorrectableVS and uncorrectableVD, curious what does `V` mean here?

Reply:
the v refers to the volatile keyword associated with these errors.